### PR TITLE
Enable more podman-remote pod commands

### PR DESCRIPTION
**Pull request by baude**
enable pod start, stop, and kill subcommands for the remote-client.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/adapter/pods.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		podids []string
+	)
+	pods, err := shortcuts.GetPodsByContext(cli.All, cli.Latest, cli.InputArgs, r.Runtime)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+	for _, p := range pods {
+		killed := true
+		conErrs, stopErr := p.Kill(signal)
```
Comment (TomSweeneyRedHat) [inline pkg/adapter/pods.go]: nit, should this bet stopErr or killErr?

File: test/e2e/pod_stop_test.go [hunk 2]
```diff
@@ -137,6 +137,5 @@ var _ = Describe("Podman pod stop", func() {
 		session = podmanTest.Podman([]string{"pod", "stop", podid1, "doesnotexist"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(125))
-		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(0))
```
Comment (TomSweeneyRedHat) [inline test/e2e/pod_stop_test.go]: should this still hold?

**Comment by TomSweeneyRedHat** [comment]
@baude merge and test fun still.

File: test/e2e/pod_stop_test.go (hunk 2 above)
Comment (baude) [reply test/e2e/pod_stop_test.go]: the behaviour has changed slightly

Comment (mheon) [reply test/e2e/pod_stop_test.go]: How so? We should be killing everything, including the infra container

Comment (baude) [reply test/e2e/pod_stop_test.go]: the way the error handling is being done now wrt to BOGUS pod names is if the input has bogus pod identifiers in it, we will return an error first.  it used to be we tried a brute force regardless.

**Comment by rhatdan** [comment]
LGTM

**Comment by baude** [comment]
if someone can merge me, I have PRs stacked up to add!

**Pull request merged**